README.md

```diff
@@ -50,6 +50,12 @@ c.pushMetric('multiObjectDelete', '3d534b1511e5630e68f0', {
 });
 ```
 
+If an error occurs during and the client is unable to record metrics in the
+underlying datastore, metric data is instead stored in a local cache. Utapi
+attempts to push these cached metrics every five minutes, by default. The
```

Comment:
I've added an explanation here. However, there is no special alert action for such a case, aside from the regular standard behavior of UtapiClient logging an `InternalError` and `UtapiReplay` logging the actions.